include/freebsd/osd.h

```diff
@@ -76,6 +76,11 @@ static inline size_t ofi_ifaddr_get_speed(struct ifaddrs *ifa)
 	return 0;
 }
 
+static inline int ofi_ifaddr_get_mtu(const struct ifaddrs *ifa)
+{
+	return -1;
```

Comment:
Why not just return 1500 as the default?  Setting mtu = -1 as a default looks like a recipe for problems and missed error checking

Reply:
Thanks for your feedback. One reason for using the value -1 is that it removes the need to put default MTU size values in a bunch more files. If there is there a good, single place to put such a definition, I must have missed it. I'm open to suggestions.

Reply:
I actually disagree with Sean for once. Returning -1 is similar to an enosys indicating that finding the MTU size is not supported/possible. Returning 1500 seems like it's falsely stating that it was able to determine that to be the MTU size. I think it's more appropriate for the default to be in the caller where it defaults to 1500 if were unable to find it.

Reply:
Then you need to add checks everywhere that the mtu size is valid prior to using it.  There are several places where mtu == -1, but that value is used anyway, with, maybe, checks coming later that the calculation is valid.  If the mtu can't be determined, then remove that interface from the list.  I can't tell that these changes work at all on windows.

Reply:
Aaaand back to agreeing with Sean. Yes, proper error checking is absolutely necessary for that to work.

Reply:
Wouldn't removing an interface whose mtu can't be determined potentially break existing client code? Doesn't the code in the main branch implicitly assume mtu=1500 for any interface supporting udp to determine the endpoint `max_msg_size` and `inject_size`? Falling back to that behavior seems like it might be safer (from that perspective) than removing the interface from the list. From a correctness point of view, dropping the interface seems preferable. For my use case, either choice is fine, so I could use some advice about how to proceed.